doc-Infrastructure_Migration_Solution_Guide/cfme/IMS_1.1/modules/proc_Installing_rhv_4_2.adoc

```diff
@@ -21,11 +21,13 @@ For details, see https://access.redhat.com/articles/417343[Ports used by Red Hat
 [NOTE]
 ====
 IMS only supports shared storage, such as NFS, iSCSI, or FCP. Local storage is not supported.
+
+Although the ISO storage domain has been deprecated, it is required for migration.
```

Comment:
My only concern here is the link through below to the RHV topic, where it states that ISO storage domain is deprecated, proceeds to instruct users to upload ISO images to a data domain, then reverts back to using ISO storage domain as the location. It gets a little confusing, but if the admins performing theses tasks are knowlegable on the concepts I guess it shouldn't cause much trouble

Reply:
ISO domain is deprecated for RHV but required for IMS. That's why I added the note. As long as RHV provides documentation for adding ISO domains and uploading to ISO domains, I'll have to link to them. When RHV stops supporting this, the documentation will have to be moved to IMS.